argocd/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM alpine as build
+# renovate: datasource=github-releases depName=argoproj/argo-cd
+ENV VERSION=2.10.2
+ARG TARGETARCH
+RUN if [["$TARGETARCH" = "amd64"]]; then \
+      export CHECKSUM=a8da9fa1ea7b7072007f535d39526ad0c4a8b8eb58779b48774126335c576187; \
+    elif [["$TARGETARCH" = "arm64"]]; then \
+      export CHECKSUM=8314c9fda218145344cb0af68ceb8f31338bf11ab4ec315ba553cf6268e17444; \
+    fi && \
```

```suggestion
RUN if [[ "$TARGETARCH" = "amd64" ]]; then \
      export CHECKSUM=a8da9fa1ea7b7072007f535d39526ad0c4a8b8eb58779b48774126335c576187; \
    elif [[ "$TARGETARCH" = "arm64" ]]; then \
      export CHECKSUM=8314c9fda218145344cb0af68ceb8f31338bf11ab4ec315ba553cf6268e17444; \
    fi && \
```

otherwise `earthly --no-cache +build --BUNDLE=argocd` fails with:

```
               build | /bin/sh: [[amd64: not found
               build | /bin/sh: [[amd64: not found
```